liferay-frontend/liferay-portal
# LPS-141230 Test that verifies a Custom Element can be displayed in a portlet

JIRA Ticket: https://issues.liferay.com/browse/LPS-141230

**Test Scenario**:

1. Create a Custom Element Widget referencing vanilla counter custom element
2. Custom Element Widget on content page
3.  Navigate to content page
4. See the Vanilla Counter is displayed
